tools/ci_build/github/azure-pipelines/templates/linux-ci.yml

```diff
@@ -2,7 +2,7 @@ parameters:
   AgentPool : 'Linux-CPU'
   JobName : 'Linux_CI_Dev'
   SubmoduleCheckoutMode: ''
-  RunDockerBuildArgs: '-o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--use_mklml --use_tvm --build_wheel"'
+  RunDockerBuildArgs: '-o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--build_wheel"'
```

Comment:
just confirming, we also are removing use_tvm ?

Reply:
We have a separated CI build pipeline just for nuphar. TVM is only needed for nuphar, so we don't need to put it here.